web/src/features/modals/AnnouncementModal.tsx

```diff
@@ -0,0 +1,83 @@
+import { SourceColorAnnouncementImage } from 'icons/announcementSourceColorImage';
+import { useAtom } from 'jotai';
+import { atomWithStorage } from 'jotai/utils';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { HiXMark } from 'react-icons/hi2';
+import { hasOnboardingBeenSeenAtom } from 'utils/state/atoms';
+
+const hasAnnouncementBeenSeenAtom = atomWithStorage(
+  'announcementSeen',
+  localStorage.getItem('announcementSeen') ?? false
+);
+
+const handleTierGaterClick = (event: React.MouseEvent<HTMLDivElement>) => {
+  event.stopPropagation();
+  event.preventDefault();
+};
+
+export default function AnnouncementModal() {
+  const { t } = useTranslation();
+  const [hasAnnouncementBeenSeen, setHasAnnouncementBeenSeen] = useAtom(
+    hasAnnouncementBeenSeenAtom
+  );
+  const [hasOnboardingBeenSeen] = useAtom(hasOnboardingBeenSeenAtom);
```

Comment:
We should also support the `skip-onboarding` query parameter that some people use for embedding the map on dashboards. See past example here: https://github.com/electricitymaps/electricitymaps-contrib/pull/5840/files#diff-0f9eeb8600aad95a9ee44374749ce1d84315dd3e062985528e002e29a688fb21R62-R64